recipe/meta.yaml

```
@@ -26,17 +26,36 @@ requirements:
     - mako
     - parse
     - parse_type
-    - pytest >=6.2.0
+    - pytest >=7.0.0
     - typing-extensions
+    - packaging
+    - gherkin-official >=29.0.0
```

```suggestion
    - gherkin-official >=29.0.0,<30.0.0
```
Because of https://github.com/pytest-dev/pytest-bdd/blob/8.1.0/pyproject.toml#L44C21-L44C28